internal/template-validator/tlsinfo/tlsinfo.go

```diff
@@ -192,8 +193,8 @@ func (ti *TLSInfo) CreateTlsConfig() *tls.Config {
 	}
 
 	if !ti.sspTLSOptions.IsEmpty() {
-		tlsConfig.CipherSuites = ti.sspTLSOptions.CipherIDs(nil)
-		minVersion, err := ti.sspTLSOptions.MinTLSVersionId()
+		tlsConfig.CipherSuites = common.CipherIDs(ti.sspTLSOptions.OpenSSLCipherNames, &logger.Log)
```

Comment:
Please keep the `nil` log here, because the change is not part of this PR. If you want, you can open a new PR to add the log.